backend/package.json

```diff
@@ -45,6 +45,7 @@
     "start": "NODE_ENV=production node ./build/index.js",
     "test": "cross-env NODE_ENV=test jest --verbose --runInBand",
     "test:watch": "jest --watchAll",
+    "pretest":"DATABASE_URL=postgres://test:test@localhost:5432/test node-pg-migrate -m ./src/db/migrations up",
```

Comment:
Our discussed pretest didn't like working in github actions

Reply:
let's do scripts as discussed: citest for gh and test for local tests